<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CLMessageBox</class>
 <widget class="QDialog" name="CLMessageBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>412</width>
    <height>244</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LMessageBox</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border-top-color: rgb(255, 0, 255);
background-color: rgb(255, 255, 255);
border-radius:6px</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true">QFrame
{
	background-color:#ffffff;
	border:none;
	border-radius:10px;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>16</number>
      </property>
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="topMargin">
         <number>10</number>
        </property>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="labelText">
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 14px;
color:rgba(51,51,51,1);
font-weight:400;</string>
          </property>
          <property name="text">
           <string>text</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="spacing">
         <number>15</number>
        </property>
        <property name="leftMargin">
         <number>45</number>
        </property>
        <property name="topMargin">
         <number>15</number>
        </property>
        <property name="rightMargin">
         <number>45</number>
        </property>
        <property name="bottomMargin">
         <number>15</number>
        </property>
        <item>
         <widget class="QPushButton" name="btnCancel">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>28</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
	width:140px;
	height:34px;	
	background:rgba(239,242,248,1);
	border-radius:3px;	
	font-size:14px;
	color:#3f3f3f;
	font-weight:400;
}
QPushButton:hover {
    color: rgba(97,154,255,1);
}

QPushButton:pressed{  
background: #6666FF;
}  

</string>
          </property>
          <property name="text">
           <string>取消</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnOK">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>28</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
	width:140px;
	height:34px;
	background:rgba(97,154,255,1);
	border-radius:3px;
	font-size:14px;
	color:rgba(255,255,255,1);
	font-weight:400;
}
QPushButton:hover {
	background-color: rgb(103,171,240);
}
QPushButton:pressed{  
background: #6666FF;
}  
</string>
          </property>
          <property name="text">
           <string>确定</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
